epidose/device/update_filter_d.sh

```diff
@@ -64,6 +64,7 @@ while : ; do
       log "Will retry in $WIFI_RETRY_TIME s"
       sleep "$WIFI_RETRY_TIME"
     else
+      check_for_updates
```

Comment:
Why is this in the `else` block of the `get_new_filter` condition?  Execute it in all cases.

Reply:
The program executes the `if` block if there is no Internet connection.
In that case, `git pull` will not work.
Are you sure that I should add the `check_for_updates` there too?

Reply:
It executed the `then` block if getting the filter has failed, no?  This is not directly related to the availability of an Internet connection.

Reply:
True, the server might be done!

Reply:
Can't you put `check_for_updates` just once before the `if`?